### PR TITLE
avoid flush repeatedly when has finished flushing

### PR DESCRIPTION
**Pull request by jackjoesh**
Modify content： avoid flush repeatedly when has finished flushing.  
After my testing ,this modification can save cpu usage in high concurrency service.

### CODE REVIEW

File: src/com/esotericsoftware/kryo/io/OutputChunked.java [hunk 1]
```diff
@@ -58,8 +58,9 @@ public void flush () throws KryoException {
 			} catch (IOException ex) {
 				throw new KryoException(ex);
 			}
+		}else{
```
Comment (theigl) [inline src/com/esotericsoftware/kryo/io/OutputChunked.java:61]: Please apply project formatting to your changes. There should be whitespace around the `else`.

**Review by theigl — CHANGES_REQUESTED** [review]
Apart from minor formatting, this PR looks good to me.

**Comment by theigl** [comment]
Thanks @jackjoesh! I think this change makes sense and I will apply it.

File: src/com/esotericsoftware/kryo/io/OutputChunked.java (hunk 1 above)
Comment (jackjoesh) [reply src/com/esotericsoftware/kryo/io/OutputChunked.java]: thank you , i have formtted

**Comment by jackjoesh** [comment]
> Thanks @jackjoesh! I think this change makes sense and I will apply it.

thank you for checking

**Comment by theigl** [comment]
Thanks again @jackjoesh!

**Pull request merged**